includes/plugins/wc-memberships/class-metering.php

```diff
@@ -270,5 +279,18 @@ public static function is_logged_in_metering_allowed( $post_id = null ) {
 	public static function is_metering() {
 		return self::is_frontend_metering() || self::is_logged_in_metering_allowed();
 	}
+
+	/**
+	 * Store the article view activity push for use in the frontend metering
+	 * strategy.
+	 *
+	 * @param array $activity Activity data.
+	 *
+	 * @return array
+	 */
+	public static function get_article_view( $activity )  {
```

Comment:
@miguelpeixe the linting error is here. two spaces after the parenthesis. Now I'm trying to understand why phpcs is breaking

Reply:
Ah ha! 🙇